app/controllers/replies_controller.rb

```diff
@@ -235,6 +235,7 @@ def restore
     end
 
     new_reply = Reply.new(audit.audited_changes)
+    new_reply.created_at = Audited::Audit.find_by(action: 'create', auditable_id: params[:id]).created_at
```

Comment:
Is it possible we'd have more than one of these (if e.g. we'd restored it before?)